test/autonomous-admin/DecentAutonomousAdminV1.test.ts

```diff
@@ -0,0 +1,140 @@
+import { SignerWithAddress } from '@nomicfoundation/hardhat-ethers/signers';
+import { expect } from 'chai';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import { ethers } from 'ethers';
+import hre from 'hardhat';
+import {
+  DecentAutonomousAdminV1,
+  DecentAutonomousAdminV1__factory,
+  MockHats,
+  MockHats__factory,
+  MockHatsElectionsEligibility,
+  MockHatsElectionsEligibility__factory,
+} from '../../typechain-types';
+import { topHatIdToHatId } from '../helpers';
+import { currentBlockTimestamp, setTime } from '../time';
+
+describe('DecentAutonomousAdminHatV1', function () {
```

Comment:
Hope one day we'll migrate to `Foundry` / `Forge`...

Reply:
me too